docs/leap_sapi/index.rst

```diff
@@ -145,4 +145,40 @@ Problem Storage
 
 Up to 1000 of your most recent problems are stored and accessible in the Leap
 service for up to 365 days; if the number of your problems exceeds 1000, the
-Leap service begins to delete the oldest ones.
\ No newline at end of file
+Leap service begins to delete the oldest ones.
+
+.. _deprecated_features:
+
+Feature Deprecation
+===================
+
+A Leap service feature may be deprecated and, during the deprecation period,
+information about such a deprecation is displayed whenever the feature is used.
+For example, a message would be displayed when a deprecated SAPI endpoint is
+called. For the Ocean SDK, deprecation messages are displayed in a terminal; for
+SAPI, deprecation messages are displayed as HTTP ``X-Deprecation`` response
+headers. A deprecation message includes the following:
+
+*   Brief description
```

Comment:
I think that similar to your description of ``context``, the format of this part of the message needs to be described too, no?